migrate2rocky/migrate2rocky.sh

```diff
@@ -188,8 +188,11 @@ exit_clean () {
 }
 
 pre_check () {
-    if [[ -e /etc/rhsm/ca/katello-server-ca.pem ]]; then
-	exit_message "Migration from Katello-modified systems is not supported by migrate2rocky."
+    if [[ -e /etc/yum.repos.d/redhat.repo ]]; then
+	exit_message "Migration from Katello-modified systems is not supported by migrate2rocky.\nPlease unconfigure all Katello configured repos, remove the /etc/yum.repos.d/redhat.repo file and re-enable the original repos."
```

Comment:
\n doesn't work in this context, you need to change the quoting style to $'' for \n to work.
That said, I don't like this change, I would rather have it say, "Migration from katello-modified systems is not supported by migrate2rocky.  See the README file for details." and then put any extra details in the README.md.  I would also like to re-iterate (in the README.md) that such a migration is not supported and should it blow up your system you're on your own.

The other issue is that RHEL 8 systems (without katello) have the file you're checking for, and that migration *is* supported and works without issue, so the test for redhat.repo won't work here and cannot be included because we will be bailing on a supported migration.

Reply:
Okay sorry for the new issues, is the new version better?
I'm now checking first if Katello or Uyuni is even configured and if it will check against the repo files, I'm not sure how this in case of RHEL then... I hope this narrows down the possible side effects.